cea/utilities/helpers.py

```diff
@@ -4,251 +4,8 @@
-    """
-    hour relative to start of heating season to hour of year
-
-    :param seasonhour: hour relative to start of heating season
-    :type seasonhour: int
-    :returns hoy: hour of year
-    :rtype hoy: int
-    """
-
-    hoy_heat_stop, hoy_heat_start = gv.seasonhours
-
-    # convert negative seasonhours to positive (for heating up phase consideration)
-    if seasonhour < 0:
-        seasonhour = 8760 - abs(seasonhour)
-
-    if check_hoy(seasonhour):
-        if seasonhour < 8760 - hoy_heat_start:
-            hoy = hoy_heat_start + seasonhour
-        else:
-            hoy = hoy_heat_start - 8760 + seasonhour
-        return hoy
-
-    else:
-        return None
-
-
-def check_hoy(hoy):
-    """
-    check for hour of year within bounds
-    hoy: hour of year
-    bool
-    """
-
-    if 0 <= hoy <= 8759:
-        return True
-    else:
-        print('Error: hour of year out of bounds')
-        print(hoy)
-        return False
-
-
-def check_doy(doy):
-    """
-    check for day of year within bounds
-    doy: day of year
-    bool
-    """
-
-    if 0 <= doy <= 359:
-        return True
-    else:
-        print('Error: day of year out of bounds')
-        print(doy)
-        return False
-
-
-def is_nighttime_hoy(hoy):
-    """
-    Check if a certain hour of year is during night or not
-    hoy: hour of year
-    bool
-    """
-    if check_hoy(hoy):
-
-        location = 'CH'  # TODO get location
-
-        start_night = 21  # 21:00 # TODO: make dynamic (e.g. as function of location/country)
-        stop_night = 7  # 07:00 # TODO: make dynamic (e.g. as function of location/country)
-
-        if start_night <= hoy_2_hod(hoy) or stop_night >= hoy_2_hod(hoy):
-            return True
-        else:
-            return False
-    else:
-        return
-
-
-def is_daytime_hoy(hoy):
-    """
-    Check if a certain hour of the year is during the daytime or not
-    hoy : hour of year
-    bool
-    """
-    if check_hoy(hoy):
-
-        if not is_nighttime_hoy(hoy):
-            return True
-        else:
-            return False
-    else:
-        return
-
-
-def is_heatingseason_hoy(hoy):
-    """
-    checks if a certain hour of the year is part of the heating season or not
-    hoy : hour of year
-    bool
-    """
-
-    if check_hoy(hoy):
-
-        #TODO: get location from (?)
-        location = 'CH'
-
-        #TODO: get start heating season f(location)
-        seasonhours = [3216, 6192]
-
-        if hoy > seasonhours[1] or hoy < seasonhours[0]:
-            return True
-        else:
-            return False
-
-    else:
-        return
-
-
-def is_coolingseason_hoy(hoy):
-    """
-    checks if a certain hour of the year is part of the cooling season or not
-    hoy : hour of year
-    bool
-    """
-
-    if check_hoy(hoy):
-        if not is_heatingseason_hoy(hoy):
-            return True
-        else:
-            return False
-
-    else:
-        return
-
 
 def sind(angle):
```

Comment:
maybe we can move this closer to home...

Reply:
don't know who is using that.
I think @shanshanhsieh in solar radiation calculation.
Usually I don't try to interfere.

Reply:
couldn't find any usages in the whole project so removed it.